Dockerfile

```diff
@@ -16,7 +14,6 @@ RUN apt-get update && \
 	rm -rf /var/lib/apt/lists/*
 
 COPY . /usr/src/app/
-RUN flit install -s
```

Comment:
Her in the place of `RUN flit install -s` we should put `RUN python setup.py develop`

Reply:
@rochacbruno 
done!